src/main/webapp/styles/common.css

```diff
@@ -111,8 +111,3 @@ nav ul li a {
   font-size: 18px;
   text-decoration: none;
 }
-
-.content {
-  text-align: center;
-  min-height: calc(100vh - 259px);
-}
```

Comment:
This will be added later, when we merge, as we have some content differences between us that we need to work out. There are a couple of conflicts, but they'll be sorted out over the next few commits.

Reply:
Sounds good. Thanks!